internal/oci/oci_wasi_tinygo.go

```diff
@@ -0,0 +1,18 @@
+//go:build wasip1 || wasip2 || tinygo
+
+package oci
+
+import (
+	"bytes"
+	"context"
+	"errors"
+)
+
+// IsOCIPath checks if a given path is an OCI path
+func IsOCIPath(path string) bool {
```

Comment:
Can this be implemented once? Does package `github.com/regclient/regclient/types/ref` import anything unsupported on TinyGo or WASI?

Reply:
Yes, good catch